libraries/botbuilder-ai/tests/qna/test_qna.py

```diff
@@ -195,19 +187,19 @@ def test_options_passed_to_ctor(self):
             options=options
         )
 
-        first_answer = result['answers'][0]
+        first_answer = result[0]
         has_at_least_1_ans = True
-        first_metadata = first_answer['metadata'][0]
+        first_metadata = first_answer.metadata[0][0]
 
         # Assert
         self.assertIsNotNone(result)
-        self.assertEqual(has_at_least_1_ans,  len(result) >= 1 and len(result) <= options.top)
-        self.assertTrue(question in first_answer['questions'])
-        self.assertTrue(first_answer['answer'])
-        self.assertEqual('is a movie', first_answer['answer'])
-        self.assertTrue(first_answer['score'] >= options.score_threshold)
-        self.assertEqual('movie', first_metadata['name'])
-        self.assertEqual('disney', first_metadata['value'])
+        # self.assertEqual(has_at_least_1_ans,  len(result) >= 1 and len(result) <= options.top)
```

Comment:
Just a question, is there any reason for keeping this comment?

Reply:
Whoops, meant to delete, not comment out

I'll remove it! :)